conda/recipes/ucxx/meta.yaml

```
@@ -59,7 +64,10 @@ outputs:
       host:
         - ucx
       run:
+        {% if cuda_major == "11" %}
         - cudatoolkit {{ cuda_spec }}
```

We're using `cuda-version` to constrain `cudatoolkit` for CUDA 11 packaging.
```suggestion
        - cudatoolkit
```

---

I based this off of an older cuDF review in my local environment, apologies for that. This is and others below are now fixed.

---

Done in commit ( https://github.com/rapidsai/ucxx/pull/61/commits/44cab6ffa3eb2609a7341036a0cb5da17781f1b3 )